plugins/coverbrowser.koplugin/main.lua

```diff
@@ -1,5 +1,6 @@
 local InputContainer = require("ui/widget/container/inputcontainer")
 local UIManager = require("ui/uimanager")
+local InputDialog = require("ui/widget/inputdialog")
```

Comment:
This would have to go above UIManager, see <https://github.com/koreader/koreader-base/wiki/Coding-style#order-of-requires>.

(More of a generic comment; not entirely sure about this commit in and of itself.)